src/main/java/teammates/storage/api/FeedbackResponseMonitorDb.java

```diff
@@ -37,4 +44,28 @@ FeedbackResponseRecordAttributes makeAttributes(FeedbackResponseRecord entity) {
 
         return FeedbackResponseRecordAttributes.valueOf(entity);
     }
+
+    /**
+     * Gets a set of response records with chosen duration and interval
+     */
+    public Set<FeedbackResponseRecord> getResponseRecords(long duration, long interval) {
+        long currentTimeInSec = System.currentTimeMillis() / 1000;
+        long startTimeInSec = currentTimeInSec - duration;
+        List<Key<FeedbackResponseRecord>> keysOfRecords = load().keys().list();
```

Comment:
Do the following instead to prevent two passing:
```
Set<FeedbackResponseRecord> records = new HashSet<>();
load().keys().forEach(k -> {
    // you have the key here, do the splitting
    records.add(new FeedbackResponseRecord(key.getName()));
})
```

Reply:
I need to keep ```currentTimestamp``` (which is not final) to filter the records, so I cannot put this inside the lambda

Reply:
I see. Can you avoid it then :)
Otherwise it is ok as of now.